subsys/bluetooth/host/hci_core.c

```diff
@@ -331,6 +331,37 @@ int bt_hci_cmd_send_sync(uint16_t opcode, struct net_buf *buf,
 	return 0;
 }
 
+int bt_hci_le_rand(void *buffer, size_t len)
+{
+	struct bt_hci_rp_le_rand *rp;
+	struct net_buf *rsp;
+	int count, err;
```

Comment:
Sorry if this comes across as overly pedantic, but I just realized that size_t is the right type for count as well, if you look at what others variables it interacts with (eg the result of MIN() is size_t as is the third parameter to memcpy())

Reply:
Absolutely, fixed